vars/getTemplateAppComponentConfigs.groovy

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/groovy
+
+import groovy.json.JsonSlurperClassic
+
+def call(componentConfigs, configGitRepo, configGitRef, tryMasterOnFail = false) {
+    node('nodejs') {
+        step([$class: 'WsCleanup'])
+
+        try {
+            checkoutGitRepo {
+                repoUrl = configGitRepo
+                branch = configGitRef
+                shallow = true
+            }
+        } catch (Exception e) {
+            if (tryMasterOnFail) {
+                checkoutGitRepo {
+                    repoUrl = configGitRepo
+                    branch = 'master'
+                    shallow = true
+                }
+            } else {
+                throw e
+            }
+        }
+
+        //ToDo Check this into template apps and remove this
```

Comment:
Is this TODO expected to be done in this PR? If not, it might be good to create a JIRA issue in the FH project to get it done, and link that here in the comment. What do you think?